### ISSUE TITLE
Luarocks install/use (on top of OpenResty) fails

### ISSUE THREAD

**Issue by distortedsignal** [luarocks/luarocks#1109]
I'm trying to use OpenResty in some very particular ways, and I would like to use luarocks to manage my packages on top of OpenResty. As such, I installed OpenResty, and then attempted to install luarocks on top of it.

```bash
> yum install -y openresty openresty-resty
...

> wget https://luarocks.org/releases/luarocks-3.2.0.tar.gz
...

> tar zxpf luarocks-3.2.0.tar.gz
...

> cd luarocks-3.2.0
...

> ./configure --with-lua=/usr/local/openresty/luajit

Configuring LuaRocks version 3.2.0...

Lua version detected: 5.1
Lua interpreter found: /usr/local/openresty/luajit/bin/luajit
lua.h found: /usr/local/openresty/luajit/include/luajit-2.1/lua.h
unzip found in PATH: /usr/bin

Done configuring.

LuaRocks will be installed at......: /usr/local
LuaRocks will install rocks at.....: /usr/local
LuaRocks configuration directory...: /usr/local/etc/luarocks
Using Lua from.....................: /usr/local/openresty/luajit

* Type make and make install:
  to install to /usr/local as usual.
* Type make bootstrap:
  to install LuaRocks into /usr/local as a rock.

> make bootstrap
mkdir -p "/usr/local/etc/luarocks"
install -m 644 "build/config-5.1.lua" "/usr/local/etc/luarocks/config-5.1.lua"
./luarocks make --tree="/usr/local"

No existing manifest. Attempting to rebuild...
luarocks 3.2.0-1 is now installed in /usr/local (license: MIT)
```

But it seems like there is still some issue using the tool -

```bash
> luarocks
/usr/local/openresty/luajit/bin/lua: /usr/local/share/lua/5.1/luarocks/fs.lua:79: cfg is not initialized, please run cfg.init() first
stack traceback:
	[C]: in function 'error'
	/usr/local/share/lua/5.1/luarocks/fs.lua:79: in function 'init'
	/usr/local/share/lua/5.1/luarocks/cmd.lua:447: in function 'run_command'
	...cal/lib/luarocks/rocks-5.1/luarocks/3.2.0-1/bin/luarocks:35: in main chunk
	[C]: at 0x00404bb0

> luarocks install luasocket
/usr/local/openresty/luajit/bin/lua: /usr/local/share/lua/5.1/luarocks/fs.lua:79: cfg is not initialized, please run cfg.init() first
stack traceback:
	[C]: in function 'error'
	/usr/local/share/lua/5.1/luarocks/fs.lua:79: in function 'init'
	/usr/local/share/lua/5.1/luarocks/cmd.lua:447: in function 'run_command'
	...cal/lib/luarocks/rocks-5.1/luarocks/3.2.0-1/bin/luarocks:35: in main chunk
	[C]: at 0x00404bb0
```

Is this a known issue? Is there something goofy with my lua install?

**Comment by distortedsignal** [comment]
Ok, I wasn't totally honest about how I installed openresty - it went more like this:

```bash
yum install openresty openresty-resty
...
ln -s /usr/local/openresty/luajit/bin/luajit /usr/local/openresty/luajit/bin/lua
...
make bootstrap
```

Sorry for the initial confusion.

**Comment by distortedsignal** [comment]
It looks like doing `make install` works - should the docs [for the quickstart guide](https://luarocks.org/#quick-start) be changed?

**Comment by hishamhm** [comment]
@distortedsignal good call, I have updated the quick start guide.

Also, I think the installation issue you had was fixed in 3.2.1. I'm closing this for now; feel free to reopen if the problem resurfaces. Thank you!
